src/TypeScript.UnitTests/EslintBridgeClient/EslintBridgeClientTests.cs

```diff
@@ -131,11 +155,26 @@ public void Dispose_DisposesHttpWrapper()
             httpWrapper.VerifyNoOtherCalls();
         }
 
-        private TypeScript.EslintBridgeClient.EslintBridgeClient CreateTestSubject(IEslintBridgeHttpWrapper httpWrapper = null, ILogger logger = null)
+        private TypeScript.EslintBridgeClient.EslintBridgeClient CreateTestSubject(IEslintBridgeHttpWrapper httpWrapper = null, 
```

Comment:
Using the same name for a namespace and a type makes referring to the type a bit clumsy. Why not make them different? e.g. change the namespace to `EslintBridge`?

Reply:
I'll open a separate PR rename 👍